### ISSUE TITLE
ESM Support

### ISSUE THREAD

**Issue by Falven** [microsoft/ApplicationInsights-node.js#1205]
When will ESM support be available for this package?

**Comment by xwipeoutx** [comment]
Yep, just got terrifically burnt by this :( Did you find a simple workaround?

I'm seeing what I can find, will post here if I get something clean

**Comment by salmipa** [comment]
Does this really mean appinsights doesn't support ESM-modules? I have tried this and that with version 3.1.0 and nothing works.

**Comment by work933k** [comment]
By overriding a dependency, the error seems to be resolved. Maybe this also works for you?
```
...
  "overrides": {
    "@opentelemetry/instrumentation": "0.52.1"
  },
...
```
